Rakefile

```diff
@@ -2,7 +2,7 @@ require "foodcritic/version"
 require "bundler"
 require "rake/testtask"
 
-task :default => [:man, :install, :rubocop, :test, :features]
+task :default => [:man, :install, :test, :features, :rubocop]
```

Comment:
usually I sort these by how long it takes to run, so I'm a bit confused by this change